Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -789,15 +935,15 @@ namespace OpenInfraPlatform {
 					throw oip::UnhandledException(conic);
 				}
 
-				// (1/2) IfcCircle SUBTYPE OF IfcConic
+				// IfcCircle SUBTYPE OF IfcConic
 				/**********************************************************************************************/
-				/*! \brief Calculates ponts of the circle curve.
+				/*! \brief Calculates points of the circle curve.
 				* \param[in] polycurve				A pointer to data from c\ IfcCircle.
 				* \param[out] targetVec				The tessellated line.
 				* \param[out] segmentStartPoints	The starting points of separate segments.
 				* \param[in] trim1Vec				The trimming of the curve as saved in IFC model - trim at start of curve.
 				* \param[in] trim2Vec				The trimming of the curve as saved in IFC model - trim at end of curve.
-				* \param[in] senseAgreement			Does the resulting geometry have the same sense agreement as the \c IfcCurve.
+				* \param[in] senseAgreement			Does the resulting geometry have the same sense agreement as the \c IfcConic.

```

Comment:
```suggestion
				* \param[in] senseAgreement			Does the resulting geometry have the same sense agreement as the \c IfcCircle.
```

Apply similarly to other comments.

Reply:
Resolved: ffe291e